components/scream/scripts/gather_all_data.py

```diff
@@ -23,12 +23,12 @@
     "lassen" : (["module purge && module load gcc/7.3.1 cuda/10.1.243 cmake/3.14.5 spectrum-mpi netcdf/4.7.0 python/3.7.2 && export LLNL_USE_OMPI_VARS='y'"],
                   "$(which mpicxx)",
                   "bsub -Ip"),
-    "quartz" : (["module purge &&  module load StdEnv cmake/3.14.5 mkl netcdf/4.4.1.1 mvapich2/2.3 intel/19.0.4 python/3.7.2"],
+    "quartz" : (["module purge &&  module load StdEnv texlive/2016 cmake/3.14.5 mkl/2019.0 intel/19.0.4 netcdf-fortran/4.4.4 netcdf/4.4.1.1"],
```

Comment:
Do you really need texlive?

Reply:
I don't think so, but I copied the environment of modules exactly as I got it to work without running `gather-all-data`.  In truth I believe that texlive is part of the standard environment so no matter what you'll get it.  I could delete and retest if needed.

Reply:
It's not about the module being loaded, that's innocuous. The thing that worries me is that texlive 2016 may be outdated soon, and the test all may then fail because it can't find a module, and we'd have to update scripts. Might as well remove it now. And if it gets automatically loaded, that's fine.

Reply:
Fair point.  I've pushed a fix and am testing it now.  My guess is it should still be good to go.